packages/fx-core/templates/azure/provision/frontendHostingProvision.bicep

```diff
@@ -0,0 +1 @@
+Mocked frontend hosting provision module contentMocked frontend hosting provision module contentMocked frontend hosting provision module contentMocked frontend hosting provision module contentMocked frontend hosting provision module contentMocked frontend hosting provision module contentMocked frontend hosting provision module contentMocked frontend hosting provision module content
```

Comment:
this is generated by test cases, just delete this file and rebase feat/arm-change